### ISSUE TITLE
Tag template doesn't always work

### ISSUE THREAD

**Issue by Glennmen** [thepracticaldev/dev.to#1769]
**Describe the bug**
New post template URLs don't always work.

**To Reproduce**

1. Go to feed overview
2. Click on `MAKE A #GITHUNT POST` (https://dev.to/new/githunt)
3. No tag is added in the new post

**Expected behavior**
`githunt` tag should be added to new post.

**Desktop (please complete the following information):**

- OS: macOS
- Browser Chrome
- Version 72.0.3626.81

**Additional context**
Using editor v1, so wasn't sure if it was related to this issue #1073 

These URLs don't work:
https://dev.to/new/githunt
https://dev.to/new/requestforpost
https://dev.to/new/vue (not on feed page but tried manually)
https://dev.to/new/opensource (not on feed page but tried manually)

These URLs work correctly:
https://dev.to/new/help
https://dev.to/new/discuss
https://dev.to/new/explainlikeimfive
https://dev.to/new/healthydebate

**Comment by aligoren** [comment]
I'm sure this related to issue #1073 

If I use editor v1, githunt, requestforpost etc. doesn't work. New tags won't work.

I used this for test [https://dev.to/new/test](test)

I recorded the screen. I change my editor settings different tab and then I refreshed the page.

**Gif Here**

![screencast-dev to-2019 02 11-09-03-04](https://user-images.githubusercontent.com/4205423/52548087-8b061f00-2ddc-11e9-980d-548501928922.gif)
